### DISCUSSION TITLE
8.29 일일회의

### DISCUSSION THREAD

**Discussion by jlee38266** [3sam5oh/webtoon-recommendation-service#150] [일일 회의]
# Todo List

- 로컬스택을 활용한 개발 환경 구성 (ec2에 kubernetes를 설치하여 테스트 진행)
- k6 모니터링 테스트 환경에 맞게 수정
- 로컬 스택을 통해서 애플리케이션들의 설정값이 올바르게 구성이 있는지 확인
- 트러블 슈팅을 고려한 기술 또는 설정? 도입 (Readis, Thread pool)
- 추천 메소드 구현
- 인프라 마무리 필요

## 이런저런 생각
- Redis에서 다룰 수 있는 로그?... 사용자가 어떤 웹툰을 봤는가? (클릭 했는지) -> 하지만 지금은 시기 상조, 추후에 필요성이 있는지는 프로젝트를 진행하며 찾아보자
- DuckDB 써보고 싶다 (그냥 말만 해봤다)

## 개선 프로젝트 일정 관련
- 해커톤 찔러보자
- 임시 일정
- 월, 수, 금 (09:00 ~ 12:00)

**Comment by mini-xi** [comment]
만약 Redis를 사용한다면
조회 성능이 어느 정도로 나오는지 확인하고 조회 성능을 확인할 수 있는 부분에 대해 고민하고 하는게 좋을 것 같습니다

1. 사용자가 어떤 웹툰들을 조회했는지
2. 웹툰에 어떤 사용자들이 조회했는지

이런 방식들도 있을 것 같다는 생각을 사뿐히 해봤습니다
